src/internal/obj/amazon_client.go

```diff
@@ -284,6 +284,10 @@ func (c *amazonClient) Exists(ctx context.Context, name string) (bool, error) {
 	return true, nil
 }
 
+func (c *amazonClient) BucketURL() ObjectStoreURL {
+	panic("not implemented")
```

Comment:
All the other bucket URLs were implemented - is there a reason this was left out?

Reply:
There's definitely not a good reason why it was left out.  Will implement.